### ISSUE TITLE
Error: 'EasyScrollSync' is not exported by node_modules/easy-scroll-sync/dist/easy-scroll-sync.esm.js

### ISSUE THREAD

**Issue by jdromero88** [Alex61NN5/easy-scroll-sync#12]
Hey @Alex61NN5 thank you for making this library!! 

I tried to use it via NPM as a dependency and devDependency in both ways I'm getting this error:

```bash
[!] Error: 'EasyScrollSync' is not exported by node_modules/easy-scroll-sync/dist/easy-scroll-sync.esm.js, imported by src/components/Table.svelte
https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module
src/components/Table.svelte (4:11)
2:   import { onMount } from "svelte";
3:   import Icon from "./Icons.svelte";
4:   import { EasyScrollSync } from 'easy-scroll-sync';
              ^
5: 
Error: 'EasyScrollSync' is not exported by node_modules/easy-scroll-sync/dist/easy-scroll-sync.esm.js, imported by src/components/Table.svelte
```
Node version `v16.14.2`
NPM version `8.5.0`

Thanks for you help!

**Comment by Alex61NN5** [comment]
hey @jdromero88 I will fix this asap!

**Comment by Alex61NN5** [comment]
Hey @jdromero88 issue should now be fixed in v2.0.0 however import was changed from `EasyScrollSync` to `easyScrollSync`

To use in your svelte app

```
<script>
    import { easyScrollSync } from "easy-scroll-sync";
</script>

<main>
    <div data-scrollsync></div>
    <div data-scrollsync></div>
</main>
```

If you see a warning in the console you may need to reset the scroll listeners like so

```
<script>
    import { onMount } from "svelte";
    import { easyScrollSync } from 'easy-scroll-sync';
    
    onMount(() => {
        easyScrollSync();
    });
</script>
```
